subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -572,13 +575,13 @@ int hawkbit_init(void)
 	struct flash_pages_info info;
 	int32_t action_id;
 
-	fs.flash_device = DEVICE_DT_GET(FLASH_NODE);
+	fs.flash_device = STORAGE_DEV;
 	if (!device_is_ready(fs.flash_device)) {
 		LOG_ERR("Flash device not ready");
 		return -ENODEV;
 	}
 
-	fs.offset = FLASH_AREA_OFFSET(storage);
+	fs.offset = FLASH_AREA_OFFSET(STORAGE_LABEL);
```

Comment:
Should be using the macro instead

Reply:
I have intentionally created the define to use here and somehow failed to do so. Fixed now.